### PR TITLE
o [NEXUS-4780] Fix NonProxyHosts support for httpclient 4x

### PR DESCRIPTION
**Pull request by nabcos**

### CODE REVIEW

File: nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus4780/Nexus4780NonProxyHostsRemoteStorage.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import java.io.File;
+
+import org.sonatype.nexus.integrationtests.ITGroups;
+import org.sonatype.nexus.integrationtests.webproxy.AbstractNexusWebProxyIntegrationTest;
+import org.sonatype.nexus.test.utils.FileTestingUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ *
```
Comment (velo) [inline nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus4780/Nexus4780NonProxyHostsRemoteStorage.java]: Please ad JDoc

**Comment by velo** [comment]
+1

File: nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus4780/Nexus4780NonProxyHostsRemoteStorage.java (hunk 1 above)
Comment (nabcos) [inline nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus4780/Nexus4780NonProxyHostsRemoteStorage.java]: Oups, on it's way.

**Comment by nabcos** [comment]
https://builds.sonatype.org/job/nexus-oss-its-feature/267/

**Pull request merged**